From 043f7a470e32c036f0fe933ec5cd2003087d6b2e Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sat, 2 Sep 2017 23:50:42 +0200
Subject: [PATCH] CMakeLists.txt: libcgi is in C

By default, CMake checks that both C and C++ compilers are
available. However, since libcgi is only C, there's no need for a C++
compiler check. Therefore, this commit adjusts the project() variable
definition to only require C language support.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Submitted-upstream: https://github.com/rafaelsteil/libcgi/pull/38
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f98a99d..c599ca9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,7 +3,7 @@
 #
 cmake_minimum_required(VERSION 2.8.8)
 
-project(cgi)
+project(cgi C)
 string(TOUPPER ${PROJECT_NAME} PROJECT_NAME_UC)
 string(TOLOWER ${PROJECT_NAME} PROJECT_NAME_LC)
 set(LIBPREFIX ${PROJECT_NAME})
-- 
2.13.5

